olap/src/aggregator/mod.rs

```diff
@@ -344,7 +323,17 @@ mod tests {
         ctx: SessionContext,
         conclusion_feed: RecordBatch,
     ) -> anyhow::Result<impl std::fmt::Display> {
-        let conclusion_feed = ctx.read_batch(conclusion_feed)?;
+        // Setup conclusion_feed table from RecordBatch
+        let provider = MemTable::try_new(conclusion_feed.schema(), vec![vec![conclusion_feed]])?;
+        let conclusion_feed = DataFrame::new(
+            ctx.state(),
+            LogicalPlanBuilder::scan(
```

Comment:
Why do we need a separate logical plan builder for scan? What does this give us extra which the DDL does not?

Reply:
We need this in order to specify the name of the table. Otherwise it gets the `?table?` name.